src/DependencyInjection/Configuration.php

```diff
@@ -11,7 +11,7 @@ class Configuration implements ConfigurationInterface
     public function getConfigTreeBuilder()
     {
         // Maintain backwars compatibility, only merge when AWS_MERGE_CONFIG is set
-        $mergeConfig = getenv('AWS_MERGE_CONFIG') ?: false;
+        $mergeConfig = isset($_ENV['AWS_MERGE_CONFIG'])? $_ENV['AWS_MERGE_CONFIG'] : false;
```

Comment:
How would we feel about checking $_ENV first and then falling back to the getenv if it isn't there to maintain the case insensitivity? General code below to give you an idea of what I mean, not necessarily following the PSR coding guidelines:
```
if (isset($_ENV['AWS_MERGE_CONFIG'])) {
      $mergeConfig =  $_ENV['AWS_MERGE_CONFIG'];
} else {
      $mergeConfig =  getenv('AWS_MERGE_CONFIG') ?: false;
}
```

Reply:
Didn't realise `getenv` is case-insensitive on Windows.

Checking https://github.com/aws/aws-sdk-php-symfony/blob/master/composer.json#L15 has `php >= 5.5` so we can't do:

```php
$mergeConfig = $_ENV['AWS_MERGE_CONFIG'] ?? getenv('AWS_MERGE_CONFIG');
```

I'm okay with the proposed solution. I'd personally go for:
```php
if (isset($_ENV['AWS_MERGE_CONFIG'])) {
      $mergeConfig = $_ENV['AWS_MERGE_CONFIG'];
} else {
      $mergeConfig = getenv('AWS_MERGE_CONFIG');
      if ($mergeConfig) {
          trigger_deprecation('aws/aws-sdk-php-symfony', '2.5.0', 'Support for AWS_MERGE_CONFIG outside .env is deprecated and will be removed in 3.0.0.');
      }
}
```

@SamRemis you don't really need `?: false` it's already the behaviour of https://www.php.net/manual/en/function.getenv.php#refsect1-function.getenv-returnvalues

Do you need a new PR? :)

Reply:
Didn't realise `getenv` is case-insensitive on Windows.

Checking https://github.com/aws/aws-sdk-php-symfony/blob/master/composer.json#L15 has `php >= 5.5` so we can't do:

```php
$mergeConfig = $_ENV['AWS_MERGE_CONFIG'] ?? getenv('AWS_MERGE_CONFIG');
```

I'm okay with the proposed solution. I'd personally go for:
```php
if (isset($_ENV['AWS_MERGE_CONFIG'])) {
      $mergeConfig = $_ENV['AWS_MERGE_CONFIG'];
} else {
      $mergeConfig = getenv('AWS_MERGE_CONFIG');
      if ($mergeConfig) {
          trigger_deprecation('aws/aws-sdk-php-symfony', '2.5.0', 'Support for AWS_MERGE_CONFIG outside .env is deprecated and will be removed in 3.0.0.');
      }
}
```

@SamRemis you don't really need `?: false` it's already the behaviour of https://www.php.net/manual/en/function.getenv.php#refsect1-function.getenv-returnvalues

Do you need a new PR? :)

Reply:
If this PR can be updated, that's great; if not, then yes please make a new PR.  The code you wrote looks good to me as well.

Reply:
#87 ready